### ISSUE TITLE
False positive for D417 (Part 2)

### ISSUE THREAD

**Issue by PhilReinhold** [charliermarsh/ruff#1291]
Opening a new issue here to track the follow-on comment I left here:

https://github.com/charliermarsh/ruff/issues/792#issuecomment-1347239427

Transcribing that comment:

Thanks for the fix for #792! However, I've noticed that the error returns if there is more than one argument present:
```python
# test.py
def f(x, y, z):
    """Do f.

    Args:
        x: the value
        y: Another thing
        z: A final argument

    Returns: the value
    """
    return x
```

$ ruff --version
ruff 0.0.177

$ ruff --select D test.py
Found 1 error(s).
test.py:1:1: D417 Missing argument descriptions in the docstring: `y`, `z`

The error again goes away if the Returns block gets a newline. Insisting on a newline there is actually fine for me, but the confusing error message is still unfortunate.

**Comment by charliermarsh** [comment]
Thank you, sorry to miss that one.

**Comment by charliermarsh** [comment]
Going out in [v0.0.189](https://github.com/charliermarsh/ruff/releases/tag/v0.0.189).
